services/frontend/src/scenes/ProjectDetails/ProjectDetails.scss

```diff
@@ -13,7 +13,7 @@
         display: grid;
         justify-content: space-around;
         grid-template-columns: repeat(auto-fill, 100px);
-        padding: $default-spacing*1.5 0 $default-spacing*0.5 0;
+        padding: $default-spacing*1.5 $default-spacing $default-spacing*0.5 $default-spacing*1.5;
```

Comment:
Instead of having to manually set all the padding to different values why not just have 
```css
padding: $default-spacing * 1.5
```
and then change the line in `.project-contributors-badge-name` from:
```css
margin: $default-spacing 0;
```
to:
```css
margin-top: $default-spacing;
```